### PR TITLE
New ansible plugin releases (foreman_ansible, foreman_ansible_core, smart_proxy_ansible) for nightly/1.22

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly (1.22 whatever happens sooner)
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by mmoll** [comment]
@ares I guess all these require 1.22+? If yes, the `control` files need updates.

**Comment by ares** [comment]
yes, I updated the control file for ansible plugin, core does not seem to have Foreman listed (which is correct), then also update the control for smart proxy, please let me know if this is what you meant and you want me to squash in respective commits

File: plugins/smart_proxy_ansible/control [hunk 1]
```diff
@@ -10,7 +10,7 @@ XS-Ruby-Versions: all
 Package: ruby-smart-proxy-ansible
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.11.0~rc3),
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.2.0~rc1),
```
Comment (mmoll) [inline plugins/smart_proxy_ansible/control]: 2 -> 22? :)

**Comment by mmoll** [comment]
@ares one small typo, please squash the commits into the respective update commits otehrwise :+1:.

**Comment by ares** [comment]
Sorry it took me a while, it's now updated. Also pushed bump to 3.0.1 to catch up with rpm PR.

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
merged, díky @ares!

**Pull request merged**